CodeStarterCamp_Week2/main.swift

```diff
@@ -9,28 +9,37 @@
 import Foundation
 
 //Step1의 순서도를 바탕으로 로또 당첨 번호를 생성하는 함수를 생성합니다.
-var numbers: Array<Int> = Array(1...45)
-var randomNumbers = numbers.shuffled()
-func generateLottoNumbers(randomNumbers: Array<Int>) -> Set<Int> {
+func generateLottoNumbers() -> Set<Int> {
+    let numbers: Array<Int> = Array(1...45)
+    let randomNumbers = numbers.shuffled()
     let lottoNumbers: Set<Int> = Set(randomNumbers[0...5])
     return lottoNumbers
 }
 
-let lottoNumbers = generateLottoNumbers(randomNumbers: randomNumbers)
+let lottoNumbers = generateLottoNumbers()
 
 //내가 찍은 번호 배열을 myLottoNumbers 변수에 할당합니다.
 let myLottoNumbers: Set<Int> = [5, 13, 23, 25, 37, 42]
 
 //찍은 번호와 로또 당첨 번호 의 겹치는 숫자를 확인하는 함수를 생성합니다.
-func checkLottoNumbers(myLottoNumbers: Set<Int>, lottoNumbers: Set<Int>) {
-    let intersectSet = lottoNumbers.intersection(myLottoNumbers)
-    let interection: Int = intersectSet.count
-    if interection < 1 {
+func compareLottoNumbers(_ myLottoNumbers: Set<Int>, with lottoNumbers: Set<Int>) {
+    let intersectSet: Set<Int> = lottoNumbers.intersection(myLottoNumbers)
+    if intersectSet.isEmpty {
         print("아쉽지만 겹치는 번호가 없습니다.")
     } else {
-        print("축하합니다! 겹치는 번호는 \(intersectSet) 입니다!")
+        var finalAnswer = ""
+        var counter = 0
+        for number in intersectSet {
+            let numberString = String(number)
+            if intersectSet.count == counter + 1 {
+                finalAnswer = finalAnswer + numberString
+            } else {
+                finalAnswer = finalAnswer + numberString + ", "
+            }
+            counter = counter + 1
+        }
+        print("축하합니다! 겹치는 번호는 \(finalAnswer) 입니다!")
     }
 }
 
-checkLottoNumbers(myLottoNumbers: myLottoNumbers, lottoNumbers: lottoNumbers)
-
+compareLottoNumbers(_: myLottoNumbers, with: lottoNumbers)
```

Comment:
```suggestion
compareLottoNumbers(myLottoNumbers, with: lottoNumbers)
```
요기 wildCard로 선언된 argumentLabel은 생략할 수 있어요!